internal/test/wpt/Makefile

```
@@ -0,0 +1,2 @@
+test:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)